src/content/en/updates/2019/12/devtools.md

```diff
@@ -0,0 +1,232 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Support for let redeclarations in the Console, improved WebAssembly debugging, and more.
+
+{# wf_updated_on: 2019-12-05 #}
+{# wf_published_on: 2019-12-05 #}
+{# wf_tags: chrome80, devtools, devtools-whatsnew #}
+{# wf_featured_image: /web/updates/images/generic/chrome-devtools.png #}
+{# wf_featured_snippet: Support for let redeclarations in the Console, improved WebAssembly debugging, and more. #}
+{# wf_blink_components: Platform>DevTools #}
+
+# What's New In DevTools (Chrome 80) {: .page-title }
+
+{% include "web/_shared/contributors/kaycebasques.html" %}
+
+## Support for `let` redeclarations in the Console {: #let }
+
+The Console now supports redeclarations of `let` statements. The inability to redeclare
+`let` statements was a common annoyance for web developers who use the Console to experiment with
+new JavaScript code.
+
+<aside class="warning">
+  Redeclaring a <code>let</code> statement in a script will still cause a
+  <code>SyntaxError</code>. <code>let</code> redeclarations only work in the Console.
+</aside>
+
+Previously, when redeclaring a local variable with `let`, the Console would throw an error:
+
+<figure>
+  <img src="../../images/2019/12/letbefore.png"
+       alt="A screenshot of the Console in Chrome 78 showing that the let redeclaration fails."/>
+</figure>
+
+Now, the Console allows the redeclaration:
+
+<figure>
+  <img src="../../images/2019/12/letafter.png"
+       alt="A screenshot of the Console in Chrome 80 showing that the let redeclaration succeeds."/>
+</figure>
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/53a8548ce03957e7704ae88b780dfa019bf967e0 #}
+
+Chromium issue [#1004193](https://crbug.com/1004193)
+
+## Improved WebAssembly debugging {: #webassembly }
+
+DevTools has started to support the [DWARF Debugging Standard](http://dwarfstd.org/),
+which means that you can now step over code, set breakpoints, and resolve stack traces in your
```

Comment:
Hmm... note sure if / how to clarify this, but should we make a distinction that you could do all that with source maps before, but in a more limited manner?

Reply:
Changed to `increased support`